code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -487,9 +487,9 @@
 
 /mob/living/simple_animal/bot/mulebot/Moved()
 	. = ..()
-
-	for(var/mob/living/carbon/human/future_pancake in loc)
-		run_over(future_pancake)
+	if(has_gravity())
+		for(var/mob/living/carbon/human/future_pancake in loc)
+			run_over(future_pancake)
```

Comment:
does run_over affect flying mobs?

Reply:
it shouldnt affect any mobs as long as the mulebot is floating

Reply:
If the mule is not floating, does it run over mobs that are?

Reply:
IF the mulebot has WEIGHT then yeah its gonna CRUSH anyone resting on the floor which makes sense. And by flying mobs do you mean people with the xenobiology wings? if said people lie on the floor like some carpet  then they're gonna get STEAMROLLED

Reply:
huh

Reply:
If a mob is flying, will a mulebot run them over

Reply:
i guess itll crush them if it managed to move to their square,  but it cant enter their square when they're standing its path will be blocked

Reply:
Do you get what i mean? it shouldnt be an issue since the mulebot will never go to a square a human is standing on, floating or not

Reply:
Oh right that's in mobbump -> knockdown chain. separate issue i'll double check on then!